lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -95,26 +95,22 @@
 
                     // Create an event queue to track team changes
                     this.teamEvents = _.clone(Backbone.Events);
-
-                    this.teamMemberships = new TeamMembershipCollection(
-                        this.context.userInfo.team_memberships_data,
+                    this.myTeamsCollection = new MyTeamsCollection(
+                        this.context.userInfo.teams,
                         {
                             teamEvents: this.teamEvents,
-                            url: this.context.teamMembershipsUrl,
                             course_id: this.context.courseID,
                             username: this.context.userInfo.username,
-                            privileged: this.context.userInfo.privileged,
-                            staff: this.context.userInfo.staff,
-                            parse: true
+                            per_page: 10,
```

Comment:
I'm confused again. Didn't you say this was 2? Or is the server default 2, but the client is passing 10?